.circleci/config.yml

```diff
@@ -839,6 +839,10 @@ workflows:
           <<: *default-context
           requires:
             - checkout
+      - test_types_next:
+          <<: *default-context
+          requires:
+            - checkout
```

Comment:
I added the `test_types_next` job to the pipeline temporarily to check if it works. I'll remove that change before merging.

Reply:
Reverted

Reply:
Example of successful run https://app.circleci.com/pipelines/github/mui/material-ui/136902/workflows/b9c1cbb1-625e-4e3d-a68b-4da986bd0108/jobs/738050